server/csv-renderers/statistics-flow-rejections.js

```diff
@@ -14,7 +14,8 @@ module.exports = function (result, config) {
 	];
 
 	data.push.apply(data, result.map(function (row) {
-		row[0] = row[0].join(' ').replace(/\r?\n|\r/g, ' ').replace(/,/g, ' ');
+		row[0] = '\"' + row[0].join(' ').replace(/\r?\n|\r/g, ' ').replace(/"/g, "'") + '\"';
```

Comment:
In order to make it future proof (maybe some other cols added, or some suspicious sign in other cells) and clean let's use sanitation performed on this row, for all the rows.